Abudu-Samuel/htclothings-api
# #162701041 chore(database): configure mongodb

### What does this PR do?
Sets up mongo database connection

### Description of Task to be completed?

- [x] add .envsample file
- [x] add config file for mongodb instance
- [x] add mongodb connection to app.js file

### How should this be manually tested?
- Clone this repository and cd into the it
- Run `yarn` to install all needed package dependencies
- Set up local mongodb connection using [mongo installation](https://treehouse.github.io/installation-guides/mac/mongo-mac.html)  and [mongo Auth](https://docs.mongodb.com/manual/tutorial/enable-authentication/)
- Run `yarn dev:server` to start the app
- To see logs, open another terminal and run `pm2 logs`

Any background context you want to provide?

Nil

### What are the relevant pivotal tracker stories?

story type: chore
story Id: #162701041
story title: Database config

### Screenshots (if appropriate)
N/A
